src/mainboard/pcengines/apu2/mainboard.c

```diff
@@ -133,6 +134,27 @@ static void mainboard_enable(device_t dev)
 		sio_dev = dev_find_slot_pnp(0x2E, NCT5104D_GPIO1);
 		if ( sio_dev ) sio_dev->enabled = 1;
 	}
+
```

Comment:
@piotr-kleins please do not leave commented code

Reply:
@miczyg1 Sorry, I was sure I delted it. Now it is.